mmdet3d/visualization/local_visualizer.py

```diff
@@ -822,6 +822,9 @@ def add_datasample(self,
                 wait_time=wait_time)
 
         if out_file is not None:
+            # check the suffix of the name of image file
```

Comment:
It's not proper to add the suffix here, if the `out_file` is `xxx.jpg`, it can be directly used in  `mmcv.imwrite`. We can specify the format of `out_file` in docstring.